apps/asap-server/test/controllers/teams.test.ts

```diff
@@ -53,12 +54,9 @@ describe('Team controller', () => {
     identity();
   });
 
-  afterEach(() => {
-    expect(nock.isDone()).toBe(true);
-  });
-
   afterEach(() => {
     nock.cleanAll();
+    expect(nock.isDone()).toBe(true);
```

Comment:
Clear all the nock fixtures.
Test cleared nock fixtures have been completed?

Reply:
shouldn't the order matter?